code/Model/VirtualPage.php

```diff
@@ -246,7 +247,7 @@ public function getCMSFields()
                     'a',
                     [
                         'class' => 'cmsEditlink',
-                        'href' => 'admin/pages/edit/show/' . $this->CopyContentFromID,
+                        'href' => AdminRootController::get_admin_route() . '/pages/edit/show/' . $this->CopyContentFromID,
```

Comment:
```suggestion
                        'href' => $this->CopyContentFrom()->CMSEditLink(),
```
Since this is hardcoding a link to the pages admin, we can assume there's no allowance for this to be anything other than `SiteTree` - so we may as well stop hardcoding any of the link at all.